swagger-api/swagger-ui
# Decode model schema name on model component

### Description
Decodes the schema name on model 


### Motivation and Context
Fixes #5626 (for both OpenAPI v3 and Swagger V2)

### How Has This Been Tested?
- Go to `localhost:3200`
- Paste this data-url in  the url field

```
data:application/json;base64,eyJjb21wb25lbnRzIjp7InNjaGVtYXMiOnsiYVwvYitjIjp7InR5cGUiOiJudW1iZXIifX19LCJvcGVuYXBpIjoiMy4wLjMiLCJwYXRocyI6eyJcLyI6eyJnZXQiOnsicmVzcG9uc2VzIjp7ImRlZmF1bHQiOnsiY29udGVudCI6eyJhcHBsaWNhdGlvblwvanNvbiI6eyJzY2hlbWEiOnsiJHJlZiI6IiNcL2NvbXBvbmVudHNcL3NjaGVtYXNcL2F+MWIlMkJjIn19fX19fX19fQ==
```

- Expand `/` route
- Switch the default response from `example value` to `model`
- You should see `a/b+c`, without any `~1` or `%B2`

An data-uri to test Swagger v2 spec

```
data:application/yaml;base64,c3dhZ2dlcjogJzIuMCcKaW5mbzoKICBkZXNjcmlwdGlvbjogJ1N3YWdnZXIgZG9jcycKICB0aXRsZTogQVBJIGV4YW1wbGUKICB2ZXJzaW9uOiAnMS4wJwpob3N0OiBzZXJ2ZXIKYmFzZVBhdGg6IC9kb2NzCnBhdGhzOgogICcvd3MvcGVyc29uJzoKICAgIGdldDoKICAgICAgZGVzY3JpcHRpb246IEdldCBwZXJzb24KICAgICAgb3BlcmF0aW9uSWQ6ICdnZXRfcGVyc29uJwogICAgICB0YWdzOgogICAgICAgIC0gUGVyc29uCiAgICAgIHJlc3BvbnNlczoKICAgICAgICAnMjAwJzoKICAgICAgICAgIGRlc2NyaXB0aW9uOiBPawogICAgICAgICAgc2NoZW1hOgogICAgICAgICAgICB0eXBlOiBvYmplY3QKICAgICAgICAgICAgcHJvcGVydGllczoKICAgICAgICAgICAgICByZXN1bHQ6CiAgICAgICAgICAgICAgICAkcmVmOiAnIy9kZWZpbml0aW9ucy9jdXN0b20lM0ElM0FuYW1lc3BhY2UlM0ElM0FQZXJzb24nCmRlZmluaXRpb25zOgogICdjdXN0b206Om5hbWVzcGFjZTo6UGVyc29uJzoKICAgIHByb3BlcnRpZXM6CiAgICAgIGlkOgogICAgICAgIHR5cGU6IG51bWJlcgogICAgICBGaXJzdE5hbWU6CiAgICAgICAgdHlwZTogc3RyaW5nCiAgICAgIExhc3ROYW1lOgogICAgICAgIHR5cGU6IHN0cmluZwogICAgICBBZGRyZXNzOgogICAgICAgIHR5cGU6IGFycmF5CiAgICAgICAgaXRlbXM6CiAgICAgICAgICAkcmVmOiAnIy9kZWZpbml0aW9ucy9jdXN0b20lM0ElM0FuYW1lc3BhY2UlM0ElM0FBZGRyZXNzJwogICdjdXN0b206Om5hbWVzcGFjZTo6QWRkcmVzcyc6CiAgICBwcm9wZXJ0aWVzOgogICAgICBpZDoKICAgICAgICB0eXBlOiBudW1iZXIKICAgICAgc3RyZWV0OgogICAgICAgIHR5cGU6IHN0cmluZwogICAgICBjaXR5OgogICAgICAgIHR5cGU6IHN0cmluZwogICAgICAgIA==
```


### Screenshots (if appropriate):
<img width="338" alt="Screenshot 2023-02-21 at 10 15 54" src="https://user-images.githubusercontent.com/3241703/220354904-80c2be39-6436-41ac-b77d-bf10c24e9a1c.png">
Before screenshot in  #5626

### My PR contains... 

### My changes...

### Documentation

### Automated tests
